rules/core/color-hex-length/README.md

```diff
@@ -0,0 +1,27 @@
+#### 📍 color-hex-length
+
+Forces the length of hex codes used in styles to be long notation rather than short.
+
+##### 🧟 Example of incorrect code for this rule:
```

Comment:
Please change to new file structure. See https://github.com/netsells/stylelint-config-netsells/pull/17 for an example.